docs/es/contact/README.md

```diff
@@ -0,0 +1,22 @@
+---
+home: false
+# icon: home
+title: Contacto
+sidebar: false
+---
+
+## Developer Association
+
+busFaktor() e.V.  
+Wolfgang Huß, 2nd chairman  
+Haferstr. 5c  
+86179 Augsburg, Germany
+
```

Comment:
Wolfgang Huß, 2º Presidente 
Haferstr. 5c  
86179 Augsburg, Alemania

Reply:
To get the Spanish version onto the website now I allow my self to take this as a note in the follow up issue #158 .